include/CpperoMQ/Socket.hpp

```diff
@@ -46,13 +46,14 @@ class Socket
 
     auto connect(const char* address) -> void;
     auto disconnect(const char* address) -> void;
-    
+
     auto getBacklog() const                                 -> int;
     auto getHandshakeInterval() const                       -> int;
     auto getImmediate() const                               -> bool;
     auto getIoThreadAffinity() const                        -> uint64_t;
     auto getIPv6() const                                    -> bool;
     auto getLastEndpoint(size_t length, char* buffer) const -> void;
+    auto getLinger() const                                  -> int;
```

Comment:
Getting/setting linger period is available over in the `SendingSocket` mixin:

https://github.com/jship/CpperoMQ/blob/ced9ba66dfe48a12dde10aaadfa1f1da6cee99e2/include/CpperoMQ/Mixins/SendingSocket.hpp#L46

https://github.com/jship/CpperoMQ/blob/ced9ba66dfe48a12dde10aaadfa1f1da6cee99e2/include/CpperoMQ/Mixins/SendingSocket.hpp#L52

Any sockets that are `SendingSocket`s will have these member functions available to them.